From 9fd704fafbb79083889b869563120be613f885c0 Mon Sep 17 00:00:00 2001
From: mytja <mitja@severkar.eu>
Date: Sun, 27 Apr 2025 01:31:40 +0200
Subject: [PATCH 8/9] PixelPropsUtils: Refetch gms_certified_props.json on a
 more conservative basis

Fetching every 5 minutes is just useless - how frequently does the fingerprint even change?
Not every 5 minutes.
---
 .../java/com/android/server/crdroid/AttestationService.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/crdroid/AttestationService.java b/services/core/java/com/android/server/crdroid/AttestationService.java
index be8aa9248..f0e47d1f6 100644
--- a/services/core/java/com/android/server/crdroid/AttestationService.java
+++ b/services/core/java/com/android/server/crdroid/AttestationService.java
@@ -39,7 +39,7 @@ public final class AttestationService extends SystemService {
     private static final String DATA_FILE = "gms_certified_props.json";
 
     private static final long INITIAL_DELAY = 0;
-    private static final long INTERVAL = 5;
+    private static final long INTERVAL = 60;
 
     private static final boolean DEBUG = Log.isLoggable(TAG, Log.DEBUG);
     private static final boolean SPOOF_PIXEL_PI =
-- 
2.34.1

